src/fixer.js

```diff
@@ -156,6 +156,26 @@ const fixOpConcat = ({ start, fixedData, verbose }) => {
   return fixedData;
 };
 
+function fixExtraCurlyBrackets({ start, fixedData, verbose }) {
+  if (verbose) psw(chalk.magenta('Extra curly brackets'));
+
+  const targetLine = start.line - 1;
+  const fullData = fixedData.join('\n');
+  let fixedLine = removeLinebreak(fixedData[targetLine]);
+
+  const openingCount = [...fullData].filter((c) => c === '{').length;
+  const closingCount = [...fullData].filter((c) => c === '}').length;
```

Comment:
There isn't, I copied that line of code from StackOverflow without thinking about it :P. It would probably trim a couple microseconds off if it's one variable. Or maybe more since it's JavaScript.